Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
+	
+	public String getRandomWord();
```

Comment:
Not exactly. Eclipse with OpenJDK defaults to protected, so I had to add access modifiers.